src/frontend/js/pages/DashboardCourses/index.tsx

```diff
@@ -62,7 +64,16 @@ export const DashboardCourses = () => {
                 data-testid="order-enrollment-list-item"
               >
                 {isEnrollement(datum) && <DashboardItemEnrollment enrollment={datum} />}
-                {isOrder(datum) && <DashboardItemOrder order={datum} />}
+                {isCredentialOrder(datum) && <DashboardItemOrder order={datum} />}
+                {isCertificateOrder(datum) && (
+                  <HandleInDom
+                    error={
+                      new Error(
+                        'useOrdersEnrollments should only return order linked to ProductType.CREDENTIAL',
+                      )
+                    }
+                  />
```

Comment:
This is not a good practice. Currently you will spam the error handling platform as at each render you will send a new error...